lib/server/swarm.js

```diff
@@ -29,15 +29,7 @@ function Swarm (infoHash, server) {
       peer_id: peer.peerId
     }
     self._onAnnounceStopped(params, peer, peer.peerId)
-
-    // When a websocket peer is evicted, and it's not in any other swarms, close
-    // the websocket to conserve server resources.
-    if (peer.socket && peer.socket.infoHashes.length === 0) {
-      try {
-        peer.socket.close()
-        peer.socket = null
-      } catch (err) {}
-    }
+    peer.socket = null
```

Comment:
Ah, alright, thanks!

So why even nullify?, it's been deleted from the lru, once this block runs it should be gc'd